package.json

```diff
@@ -61,8 +61,8 @@
   "optionalDependencies": {
     "create-dmg": "^5.3.0",
     "dbus-next": "^0.8.2",
-    "electron-installer-debian": "^3.1.0",
-    "electron-installer-redhat": "^3.1.0",
+    "electron-installer-debian": "^3.0.0",
+    "electron-installer-redhat": "electron-userland/electron-installer-redhat#ab04829",
```

Comment:
what do you think about this? or should we wait until a new version gets published?